mars/services/task/tests/test_service.py

```diff
@@ -384,6 +384,20 @@ def _get_fields(details, field, wrapper=None):
     details = await task_api.get_tileable_details(task_id)
     assert details[r7.key]['status'] == SubtaskStatus.errored.value
 
+    def is_valid_type(value):
+        return isinstance(value, int) or isinstance(value, float) or isinstance(value, str)
```

Comment:
Multiple `isinstance`s can be simplified with a single call on tuple of types, for inst., `isinstance(value, (int, float, str))`.